static/.htaccess

```diff
@@ -8,6 +8,10 @@ Options -Indexes -MultiViews
 
 # Redirect pages - https://httpd.apache.org/docs/current/mod/mod_rewrite.html#rewriterule
 
+# Identity Authorizations
+RewriteRule ^docs/next/self-managed/identity/user-guide/groups/creating-authorizations-for-a-group/(.*)$ /docs/next/self-managed/identity/user-guide/authorizations/managing-resource-authorization/ [R=301,L]
+RewriteRule ^docs/self-managed/identity/user-guide/groups/creating-authorizations-for-a-group/(.*)$ /docs/self-managed/identity/user-guide/authorizations/managing-resource-authorization/ [R=301,L]
```

Comment:
Are these the correct target URLs? I don't see either of them on staging: 

* https://stage.docs.camunda.io/docs/next/self-managed/identity/user-guide/authorizations/managing-resource-authorization/
* https://stage.docs.camunda.io/docs/self-managed/identity/user-guide/authorizations/managing-resource-authorization/

Reply:
https://stage.docs.camunda.io/docs/next/self-managed/identity/user-guide/authorizations/managing-resource-authorizations/

Content isn't on 8.1, good catch. Also plural vs. not. Will correct.